homeassistant/components/tradfri/const.py

```diff
@@ -1,45 +1,13 @@
 """Consts used by Tradfri."""
-from typing import Final
+import logging
 
-from homeassistant.components.light import SUPPORT_TRANSITION
-from homeassistant.const import (  # noqa: F401 pylint: disable=unused-import
-    CONF_HOST,
-    Platform,
-)
+_LOGGER = logging.getLogger(__name__)
```

Comment:
Make it non protected if we want other modules to import it. And the logger should then represent the whole package and not just this module.
```suggestion
LOGGER = logging.getLogger(__package__)
```